Okta/OktaAppAuth.swift

```diff
@@ -39,6 +39,11 @@ public func login() -> Login {
     return Login()
 }
 
+public func logout() -> Logout {
+    // Logout for authorization code flow
+    return Logout(idToken: tokens?.idToken)
```

Comment:
Okta supports passing a [`state` param](https://developer.okta.com/docs/api/resources/oidc#request-parameters-4) to this endpoint. Do we want to accept an optional `state` param to support this flow?

Reply:
Passing `scope` is out of scope for now (no pun intended)